test/boost/sstable_datafile_test.cc

```diff
@@ -1035,6 +1035,66 @@ static flat_mutation_reader make_normalizing_sstable_reader(shared_sstable sst,
     return make_normalizing_sstable_reader(sst, s, query::full_partition_range);
 }
 
+SEASTAR_TEST_CASE(autocompaction_control_test) {
```

Comment:
if the nodetool now works, it would be better to add a test for the nodetool (under nodetool_additional_test.py)

Reply:
ok